openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py

```diff
@@ -44,6 +44,24 @@ class TiingoEquityHistoricalQueryParams(EquityHistoricalQueryParams):
     _frequency: Literal["daily", "weekly", "monthly", "annually"] = PrivateAttr(
         default=None
     )
+    sort: Literal[
```

Comment:
Isn't this a fruitless effort to expose this param? Sorting occurs by date, ALWAYS for this endpoint, and results are always sorted in the same direction, ascending for every provider.

Reply:
I got this error without using the sort:
`"Error: Sort column name is not a valid column"`

So, apparently the `sort` needs to be explicit in the URL.

Reply:
How come it works for me right now, without this?

Reply:
The URL appears to work fine without a sort column.

```
https://api.tiingo.com/tiingo/daily/AAPL/prices?startDate=2023-03-27&endDate=2024-03-27&resampleFreq=monthly&token=
```

![Screenshot 2024-03-27 at 10 54 01 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/7e79a4e7-090d-470c-92fb-7d02253461ac)